src/app/static/src/app/directives/translate.ts

```diff
@@ -20,7 +20,7 @@ export default <S extends TranslatableState>(store: Store<S>): DirectiveOptions
         if (attribute) {
             el.setAttribute(attribute, i18next.t(binding.value, {lng}));
         } else {
-            el.innerText = i18next.t(binding.value, {lng});
+            el.innerHTML = i18next.t(binding.value, {lng});
```

Comment:
A couple of the translated strings had links in them, e.g. https://github.com/mrc-ide/hint/pull/223/files#diff-9852cb19dfd0c9fa2b7effb126b4e7c1R120

So changed this over to innerHTML so links would be rendered.